client/src/components/Auth/Auth.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+import propTypes from 'prop-types';
+
+export default class Auth extends React.Component {
+  state = {
+    IsRedirect: false,
+    logedIn: false,
+  };
+
+  componentWillMount() {
+    const { history } = this.props;
+    fetch('api/v1/auth', {
+      method: 'POST',
```

Comment:
no need for post, since it doesn't send a body which has to be stored in database.